_episodes/39.md

```diff
@@ -0,0 +1,263 @@
+industry that append more rules and standards around technologies they can adopt
+in the stack.
+
+This is now a common practice for government agencies to use, as they are able
+to take standardized software stacks that go through rigorous validation to make
+sure the meet the standards of the government. One important element to these
+stacks are that they can be deployed in virtually any environment. A common
+way to do this is using Kubernetes and containers.
+
+### Standards and anatomy of a stack
+
+With the movement towards standardization, government contractors will generally
+build their stack using Kubernetes templates. Kubernetes underpins each of these
+stacks while telemetry, monitoring, and policy agents are layered on after that.
+For Raft, they wanted to provide a "single pane of glass" over the existing
+fragmented systems that the Department of Defense (DoD) operates on. They began
+to develop a stack that included Trino as their method to connect data over
+various silos.
+
+### Data Fabric at Raft
+
+Data Fabric is an attempt to provide government agencies the ability to set up
+a data mesh that is backed by Trino. Trino fits well in this narrative as it
+provides SQL-over-everything. Data analysts and data scientists only need to
+know SQL.
+
+> Data Fabric MVP is an end-to-end DataOps capability that can be deployed at the
+edge, in the cloud, and in disconnected environments within minutes. It provides
+a single control plane for normalizing and combining disparate data lakes, 
+platforms, silos, and formats into SQL using Trino for batch data and Apache 
+Pinot for user facing streaming analytics. 
+>
+> Data Fabric is driven by cloud native policy using Open Policy Agent (OPA) 
+integrated with Trino and Kafka to provide row and column level obfuscation. It
+provides enterprise data catalog to view data lineage, properties, and data
+owners from multiple data platforms. -- [Raft](https://datafabric.goraft.tech/)
+
+### Security concerns around Trino
+
+A common first question the Raft team gets asked is around Trino being a high
+security concern. The idea that Trino can connect to multiple data sources from
+one location brings up fear that individuals may gain access to information at
+a higher classification level than they have. The team has to educate the
+different users on the best practices and how to ensure this problem doesn't
+occur. You will need a separate deployment of Data Fabric for each 
+classification level and correctly identify policies in OPA that restrict
+visibility to information above a users' clearance.
+
+### Iron Bank container repository
+
+Iron Bank is a groundbreaking central repository of digitally-signed container 
+images, including open-source and commercial off-the-shelf software, hardened to
+the DoD’s exacting specifications. Approved containers in Iron Bank have
+DoD-wide reciprocity across all classifications, accelerating down to weeks a 
```

Comment:
accelerating the security approval process from months or even years down to weeks.